Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -89,6 +113,19 @@ class PlaceDetailViewController: UIViewController {
                         cardView.widthAnchor.constraint(equalToConstant: 343),
                         cardView.centerXAnchor.constraint(equalTo: view.centerXAnchor)]
 
+        view.addSubview(mapButton)
+        constraints += [mapButton.topAnchor.constraint(equalTo: view.topAnchor, constant: 36),
+                        mapButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 16),
+                        mapButton.heightAnchor.constraint(equalToConstant: 48),
+                        mapButton.widthAnchor.constraint(equalToConstant: 48)]
+
+        view.addSubview(mapButtonBadge)
+        constraints += [mapButtonBadge.leadingAnchor.constraint(equalTo: mapButton.trailingAnchor, constant: -10),
+                        mapButtonBadge.topAnchor.constraint(equalTo: mapButton.topAnchor),
+                        mapButtonBadge.heightAnchor.constraint(equalToConstant: 20.0),
+                        mapButtonBadge.widthAnchor.constraint(greaterThanOrEqualToConstant: 20.0)]
```

Comment:
I assume `greaterThanOrEqualToConstant` means it will wrap text size (and not unexpectedly be giant)?